packages/calcite-components/src/components/input/input.tsx

```diff
@@ -1171,9 +1171,9 @@ export class Input
     const prefixText = <div class={CSS.prefix}>{this.prefixText}</div>;
     const suffixText = <div class={CSS.suffix}>{this.suffixText}</div>;
 
-    const autofocus = this.el.autofocus || this.el.hasAttribute("autofocus") ? true : null;
-    const enterKeyHint = this.el.enterKeyHint || this.el.getAttribute("enterkeyhint");
-    const inputMode = this.el.inputMode || this.el.getAttribute("inputmode");
+    const autofocus = this.el.autofocus ? true : null;
```

Comment:
(optional) This would be even shorter, but less readable:

```suggestion
    const autofocus = this.el.autofocus || null;
```